features/CLI/CommandLine.java

```diff
@@ -0,0 +1,143 @@
+import java.io.IOException;
+import java.util.Scanner;
+
```

Comment:
Should we be importing datastructures.classes and datastructures.HelperMethods here?

Reply:
We don't need to. Since they're a part of the same package, they can create an object of the other classes without needing to import the files.